distribution/environments/build-or-push-flavors.sh

```diff
@@ -3,11 +3,15 @@ set -euo pipefail
 
 IMAGE_NAME=jenkins/evergreen
 
+set -xe
```

Comment:
I guess it's a leftover. `-e` is already enabled, and `-x` is more for debugging IMO. But well, nothing blocking.